javascript/javascript1/week2/script.js

```diff
@@ -45,29 +45,63 @@
 
 // Weather wear
 
-function recommendedWeatherWear(temperature) {
-    if (temperature <= 30 && temperature >= 20) {
-        return "Go to the beach.";
-    } else if (temperature <= 19 && temperature >= 15) {
-        return " T-shirt and shorts. ";
-    } else if (temperature <= 14 && temperature >= 7) {
-        return " Sweeter and trousers. ";
-    } else if (temperature <= 6 && temperature >= 0) {
-        return " Jacket , sweeter and trousers. ";
-    } else if (temperature <= 0 && temperature >= -15) {
-        return "Wnter- jacket, sweeter, trousers. ";
-    } else if (temperature <= -16 && temperature >= -25) {
-        return " Double- Wnter- jacket, double-sweeter, double-trousers.";
-    }
-    // else if (typeof temperature !== Number) {
-    //     return " need a number";
-    // }
-    else {
-        return " I can not help you (";
-    }
-}
+// function recommendedWeatherWear(temperature) {
+//     if (temperature <= 30 && temperature >= 20) {
+//         return "Go to the beach.";
+//     } else if (temperature <= 19 && temperature >= 15) {
+//         return " T-shirt and shorts. ";
+//     } else if (temperature <= 14 && temperature >= 7) {
+//         return " Sweeter and trousers. ";
+//     } else if (temperature <= 6 && temperature >= 0) {
+//         return " Jacket , sweeter and trousers. ";
+//     } else if (temperature <= 0 && temperature >= -15) {
+//         return "Wnter- jacket, sweeter, trousers. ";
+//     } else if (temperature <= -16 && temperature >= -25) {
+//         return " Double- Wnter- jacket, double-sweeter, double-trousers.";
+//     }
+//     // else if (typeof temperature !== Number) {
+//     //     return " need a number";
+//     // }
+//     else {
+//         return " I can not help you (";
+//     }
+// }
+
+// console.log(recommendedWeatherWear(31)) 
+
+
+
+
 
-console.log(recommendedWeatherWear(31)) 
+// Event application
+
+function getEventWeekday(beingEventDay) {
+    
+
+const weekDays = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+const currentDate = new Date().toDateString();
+
+
+const currentWeekDay = currentDate.slice(0, 3);
+
+
+    for (let index = 0; index < weekDays.length; index++) {
+        const element = weekDays[index];
+       
+        if (element === currentWeekDay) {
```

Comment:
This is a perfectly correct solution to the problem! 💯 However, I can give you an even easier solution! The object returned by `new Date()` has the following method: [`.getDay()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDay) It returns the exact index of the day that you have used in the array, meaning Sunday corresponds to 0, and Saturday to 6. This means that you can make the function much simpler! 👼